tableau-server-single-node/scripts/config-linux.sh

```diff
@@ -73,7 +73,7 @@ then
 fi
 
 # download automated-installer
-wget --remote-encoding=UTF-8 --output-document=automated-installer.sh https://raw.githubusercontent.com/tableau/server-install-script-samples/master/linux/automated-installer/automated-installer
+wget --remote-encoding=UTF-8 --output-document=automated-installer.sh https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/tableau-server-single-node/scripts/automated-installer
```

Comment:
this needs to be passed in using the _artifactsLocation parameter

Reply:
@kkader how do we do this given the ubuntuscript uri command you added?